create/templates/projects/checkout_post_purchase_next/src/index.tpl

```diff
@@ -0,0 +1,5 @@
+{{- if .Development.UsesReact -}}
```

Comment:
Gotcha. So when `{{ file` is used instead, the template engine inlines the given file without passing it through the engine, correct?
I went ahead and made the changes, but I've broken the workflow 😬. I think it might have something to do with the `.Extension.UseNext`

Reply:
I think `.` refers to the `Extension`. Have you tried just `.UsesNext`?